src/float16sr.jl

```diff
@@ -1,3 +1,4 @@
+import BFloat16s.BFloat16
```

Comment:
instead of adding this to individual files, we can add `import BFloat16s: BFloat16` to `src/StochasticRounding.jl` ?